### ISSUE TITLE
[Discharge Upgrade Wizard] [A11y] Accessibility review of DUW when it's complete

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#12802]
## Status
**[2024-09-16] [Fran]** Laura has been conducting accessibility reviews on each PR, so this ticket may be closed as a no-op.
**[2024-02-26] [Fran]** Moved to Next refinement pipeline because DUW is almost completed.
**[2024-02-26] [Fran]** Blocked by the numerous DUW implementation tickets. 

## Description

After the DUW has been built using a subtask pattern, an accessibility review needs to occur.

### User story

**AS A** Veteran, their family member, caregiver, or survivor
**I WANT** to be able to use a screen reader or other accessibility tools when stepping through the new Discharge Upgrade Wizard
**SO THAT** I'm able to fully understand the questions and answers I receive during my progress and am able to determine if I qualify for an upgrade and how to go about getting one.

## Acceptance criteria

- [ ] a11y review of the new DUW is completed
- [ ] tickets are created if any issues are found
- [ ] ticket may then be closed

**Comment by laflannery** [comment]
Unassigning myself until this is actually in a sprint

**Comment by FranECross** [comment]
Confirmed that Laura has been conducting accessibility reviews on each PR, so this ticket may be closed as a no-op. https://dsva.slack.com/archives/C52CL1PKQ/p1726513382082349?thread_ts=1726509872.681399&cid=C52CL1PKQ
